service/worker/scheduler/workflow.go

```diff
@@ -60,6 +60,8 @@ const (
 	InitialVersion SchedulerWorkflowVersion = iota
 	// skip over entire time range if paused and batch and cache getNextTime queries
 	BatchAndCacheTimeQueries
+	// use cache v2, and include ids in jitter
+	NewCacheAndJitter
```

Comment:
It seems precarious that we depend on the order of lines here since the version is serialized. I think it'd be safer if we assigned these to actual constants similar to how it's done in protobuf.

Reply:
sure